cypress/support/commands.ts

```diff
@@ -0,0 +1,54 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+
+// TODO: Pass through merchant ID, and Price, so we can test different deals?
+// Cypress.Commands.add('visitAndInjectWidget', (url, targetElement, position) => {
```

Comment:
Re: @arjun-at-roam suggestion - can you add this entire commented out piece of code to a separate PR later when you're able to implement it fully? It just seems out of scope for this PR since it's commented out 🙂